test/Microsoft.ML.AutoML.Tests/SweepableExtensionTest.cs

```diff
@@ -0,0 +1,147 @@
+            var pipeline = estimator.Append(SweepableEstimatorFactory.CreateFastForestBinary(new FastForestOption()));
+
+            pipeline.ToString().Should().Be("Unknown=>FastForestBinary");
+        }
+
+        [Fact]
+        public void AppendIEstimatorToSweepabeEstimatorPipelineTest()
+        {
+            var context = new MLContext();
+            var estimator = context.Transforms.Concatenate("output", "input");
+            var pipeline = estimator.Append(SweepableEstimatorFactory.CreateFastForestBinary(new FastForestOption()));
+            pipeline = pipeline.Append(context.Transforms.CopyColumns("output", "input"));
+
+            pipeline.ToString().Should().Be("Unknown=>FastForestBinary=>Unknown");
+        }
+
+        [Fact]
+        public void CreateSweepableEstimatorPipelineFromSweepableEstimatorTest()
+        {
+            var estimator = SweepableEstimatorFactory.CreateFastForestBinary(new FastForestOption());
+            var pipeline = estimator.Append(estimator);
+
+            pipeline.ToString().Should().Be("FastForestBinary=>FastForestBinary");
+        }
+
+        [Fact]
+        public void CreateSweepableEstimatorPipelineFromSweepableEstimatorAndIEstimatorTest()
+        {
+            var context = new MLContext();
+            var estimator = SweepableEstimatorFactory.CreateFastForestBinary(new FastForestOption());
+            var pipeline = estimator.Append(context.Transforms.Concatenate("output", "input"));
+
+            pipeline.ToString().Should().Be("FastForestBinary=>Unknown");
+
+        }
+
+        [Fact]
+        [UseApprovalSubdirectory("ApprovalTests")]
+        [UseReporter(typeof(DiffReporter))]
+        public void CreateMultiModelPipelineFromIEstimatorAndBinaryClassifiers()
+        {
+            var context = new MLContext();
+            var pipeline = context.Transforms.Concatenate("output", "input")
+                                .Append(context.Auto().BinaryClassification());
+
+            var json = JsonSerializer.Serialize(pipeline, this._jsonSerializerOptions);
+            Approvals.Verify(json);
+        }
+
+        [Fact]
+        [UseApprovalSubdirectory("ApprovalTests")]
+        [UseReporter(typeof(DiffReporter))]
+        public void CreateMultiModelPipelineFromIEstimatorAndMultiClassifiers()
+        {
+            var context = new MLContext();
+            var pipeline = context.Transforms.Concatenate("output", "input")
+                                .Append(context.Auto().MultiClassification());
+
+            var json = JsonSerializer.Serialize(pipeline, this._jsonSerializerOptions);
+            Approvals.Verify(json);
+        }
+
+        [Fact]
+        [UseApprovalSubdirectory("ApprovalTests")]
+        [UseReporter(typeof(DiffReporter))]
+        public void CreateMultiModelPipelineFromIEstimatorAndRegressors()
+        {
+            var context = new MLContext();
+            var pipeline = context.Transforms.Concatenate("output", "input")
```

Comment:
@JakeRadMSFT 

This will be essentially how user create a sweepable pipeline for multi-classification